examples.md

```diff
@@ -46,7 +46,7 @@
 Using [NuGet lock files](https://docs.microsoft.com/nuget/consume-packages/package-references-in-project-files#locking-dependencies):
 
 ```yaml
-- uses: actions/cache@v3
+- uses: actions/cache@v4
```

Comment:
https://github.com/actions/cache/pull/1378/files#diff-cfb2db6f153a4484d9bbd408f3227a264dce68fadd747cecdc30bda012b4d8cdR49 also makes this change.